src/main/scala/com/cognite/sdk/scala/v1/resources/dataPoints.scala

```diff
@@ -101,13 +102,41 @@ class DataPointsResource[F[_]](val requestSession: RequestSession[F])
       (r: Response[Unit]) => r.body
     )
 
-  /*
+  private def arrayByteSerializer(a: Array[Byte]): ByteArrayBody = ByteArrayBody(a)
+
   def insertById(id: Long, dataPoints: Seq[DataPoint]): F[Unit] =
```

Comment:
These methods were removed, and are replaced by the call to `insert(CogniteInternalId(id), dataPoints)`. Sorry for confusion with the commented out code, I wanted to leave it there to discuss what to do with them, but then we forgot about it. If you really want to add these methods back, they should only call to `insert(CogniteInternalId(id), dataPoints)`.

I guess you rather should change the `insert` method to use requestSession.post

Reply:
I agree with you, but @hakontro suggested that we keep the old method insert to avoid breaking any thing, even though I added the test for insertById and insertByExternalId, maybe we can have some new services start using these new api deployed, monitor them to make sure everything go well so that we can use `requestSession.post` inside `insert` ?
@hakontro what do you think?

Reply:
First of all I think it's a nice PR!

I think we should keep the `.insert` method, but it's fine if the implementation changes. I think Emel's suggestion is a good one, try to look for a way to reuse the code, either by having `insertById/ExternalId` call `insert`, or at least by having `insert` call `insertById`.

Reply:
thanks you all, I updated in the commit https://github.com/cognitedata/cognite-sdk-scala/pull/385/commits/fe687d16ff3fe23f9761025877616586380d8f95 :)